src/deploy/osp_deployer/profiles/xsp.json

```diff
@@ -285,7 +301,11 @@
                   "3_node"
                ]
             },
-			"powerflex_cluster_virtualip": {
+<<<<<<< HEAD
```

Comment:
Merge conflict needs attention

Reply:
Don't know why it's there. Supposed to got it resolved.